src/views/ProjectView.vue

```diff
@@ -17,7 +17,7 @@ const isPopupVisible = ref(false)
 const uniqueDataSessions = ref([])
 const newSessionName = ref('')
 const errorMessage = ref('')
-const dataSessionsUrl = store.state.dataSessionsUrl
+const apiBaseUrl = store.state.apiBaseUrl
```

Comment:
I'm losing track of the different APIs we're talking to - we may want to change this to datalabApiBase or something similar in the store. Consistency in naming is going to pay off big time here!

Reply:
lol

Reply:
yes, naming is hard